package.json

```diff
@@ -83,16 +83,17 @@
   "config": {
     "forge": {
       "packagerConfig": {
-        "asar": true
+        "asar": true,
+        "name": "Leaphy Easybloqs",
```

Comment:
Can't we default to name or productName in package.json? Minimizing places where we configure names helps consistency.
https://electron.github.io/electron-packager/master/interfaces/electronpackager.options.html#name

Reply:
Good point! Removed the redundant name and added productName to the package.json.